agents/synacc/synacc.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+import os
+import requests
+import argparse
+
+on_rtd = os.environ.get('READTHEDOCS') == 'True'
+if not on_rtd:
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import TimeoutLock
+
+
+class SynAccAgent:
+    def __init__(self, agent, ip_address, username, password):
+        self.agent = agent
+        self.lock = TimeoutLock()
+        self.ip_address = ip_address
+        self.user = username
+        self.passw = password
+
+    def get_status(self, session, params=None):
+        with self.lock.acquire_timeout(1) as acquired:
+            if acquired:
+                req = "http://" + self.user + ":" + self.passw + "@" +\
+                    self.ip_address+"/cmd.cgi?$A5"
+                r = requests.get(req)
```

Comment:
I don't think it should hold up this PR, but this could be made to support twisted by using [twisted's HTTP client](https://twistedmatrix.com/documents/current/web/howto/client.html) instead of `requests`.

Reply:
I will add this in in the next iteration that includes the acquisition proccess